Mesh_3/include/CGAL/Labeled_mesh_domain_3.h

```diff
@@ -117,15 +123,67 @@ namespace internal {
     }
   };
 
+  // Detect_features_in_domain
+  template<typename Point, typename DetectFunctor>
+  struct Detect_features_in_domain {
+    std::vector<std::vector<Point>>
+    operator()(const CGAL::Image_3& image, DetectFunctor functor) const {
+#if defined(BOOST_MSVC) && (BOOST_MSVC < 1910) //before msvc2017
+      return functor.operator()<Point>(image);
+#else
+      return functor.template operator()<Point>(image);
```

Comment:
This does not correspond to the way the functor is documented. The documentation says:
> a functor that can be passed to
> `%std::function<std::vector<std::vector<Point_3>>(const CGAL::Image_3&, Mesh_domain_with_polyline_features_3<Labeled_mesh_domain_3>&)>`